src/sources/forus-platform/js/angular-1/i18n/nl/pages/voucher-printable.pug.i18n.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    product_voucher: {
+        labels: {
+            use_for: 'Gebruik deze aanbiedings voucher voor:',
+            description: 'Dit is uw ‘{{printableTitle}}’ voucher met een QR-code',
+            show_to: 'Laat deze voucher eenmalig zien bij de destreffende aanbieder:',
+            contact_us: 'Neem dan contact op met de aanbieder.'
+        }
+    },
+    budget_voucher: {
+        labels: {
+            use_for: 'Gebruik deze budget voucher voor:',
+            description: 'Dit is uw {{printableTitle}} voucher met een QR-code',
+            show_to: 'Laat deze voucher zien bij een van de destreffende aanbieders, te vinden op:',
+            contact_us: 'Neem dan contact op.'
```

Comment:
Neem contact op met:

Reply:
I think it got fixed on my last commit:

https://github.com/teamforus/forus-frontend/pull/817/commits/232664a03a9509693f0252f040952f7a090b5ccc

https://github.com/teamforus/forus-frontend/blob/fe7d59424cdf9ea848cc3788fab8855b2859a282/src/sources/forus-platform/js/angular-1/i18n/nl/pages/voucher-printable.pug.i18n.js